com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/MtasApiServiceImpl.java

```diff
@@ -1,15 +1,18 @@
 package com.sap.cloud.lm.sl.cf.web.api.impl;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 
+import org.apache.commons.collections4.ListUtils;
 import org.cloudfoundry.client.lib.CloudControllerClient;
 import org.springframework.http.ResponseEntity;
 
+import com.google.common.collect.Lists;
```

Comment:
We're trying not to use anything from Guava, since it's just a transitive dependency coming from JClouds.